spec/components/service_unavailable_component_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe ServiceUnavailableComponent, type: :component do
```

Comment:
Not sure if this test is useful

Reply:
Makes sense to have it, even if it's minimal.